includes/post-type-functions.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Post type functions.
+ *
+ * @link       https://nataliesmith.dev
+ * @since      1.0.0
+ * @package    Poetwp
+ * @subpackage Poetwp/includes
+ */
+
+/**
+ * Register the custom post type
+ */
+function poetwp_register_post_type() {
+	register_post_type(
+		'poem',
+		array(
+			'labels'       => array(
+				'name'          => __( 'Poems' ),
+				'singular_name' => __( 'Poem' ),
```

Comment:
No text domain in i18n functions is only allowed for WP core - for plugin and themes, they should define their own text domain (and don't try to be fancy and use a constant or variable here - it needs to be a plain repeated string).

```suggestion
				'name'          => __( 'Poems', 'poetwp' ),
				'singular_name' => __( 'Poem', 'poetwp' ),
```

Reply:
Noted #16